### PR TITLE
Rework user history

### PR DESCRIPTION
**Pull request by alimtunc**
This PR offer a new way to browse your chat history.

https://github.com/Chainlit/chainlit/assets/18298759/56c6b9ed-a0ab-4e56-9b53-dd71f94f878a

### CODE REVIEW

File: frontend/src/pages/Page.tsx [hunk 1]
```diff
@@ -44,7 +45,10 @@ const Page = ({ children }: Props) => {
       {!isAuthenticated ? (
         <Alert severity="error">You are not part of this project.</Alert>
       ) : (
-        children
+        <Stack direction="row" height="100%" width="100vw" overflow="auto">
```
Comment (willydouhard) [inline frontend/src/pages/Page.tsx:48]: can't we do with 100% here? i don't like vw that much

File: frontend/src/router.tsx [hunk 2]
```diff
@@ -25,16 +23,8 @@ export const router = createBrowserRouter([
...
-      <Page>
-        <Conversation />
-      </Page>
-    )
-  },
-  {
-    path: '/dataset',
-    element: <Dataset />
+    path: '/conversations/:id?',
+    element: <Conversations />
```
Comment (willydouhard) [inline frontend/src/router.tsx:27]: shouldnt that component be named Conversation? Or ConversationPage?

File: libs/components/src/playground/chat.tsx [hunk 3]
```diff
@@ -57,7 +57,7 @@ export const ChatPromptPlayground = forwardRef(
         : 'Prompt messages';
 
     return (
-      <Stack sx={{ width: '100%' }}>
+      <Stack sx={{ width: '100%', height: '100%' }}>
```
Comment (willydouhard) [inline libs/components/src/playground/chat.tsx:60]: why are we touching the prompt playground?

File: frontend/src/pages/Page.tsx (hunk 1 above)
Comment (alimtunc) [reply frontend/src/pages/Page.tsx:48]: Actually looks like we don't need the width

**Review by willydouhard — APPROVED** [review]

**Pull request merged**